skyportal/handlers/api/allocation.py

```diff
@@ -119,6 +119,7 @@ def post(self):
 
```

Comment:
I would add a test to make sure the start date is before the end date.

Reply:
Added.